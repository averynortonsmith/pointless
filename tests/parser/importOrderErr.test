
x = 123

import "foo" as bar