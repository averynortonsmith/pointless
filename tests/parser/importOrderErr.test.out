-------------------------------------------------------------------------------
Parser Error:

Expected 'Name', got 'Import'
-------------------------------------------------------------------------------

At (line 4 column 1) in 'tests/parser/importOrderErr.test'
import "foo" as bar
^
